packages/ai/ai/src/AiModel.ts

```diff
@@ -22,88 +19,35 @@ export const TypeId: unique symbol = Symbol.for("@effect/ai/AiModel")
  */
 export type TypeId = typeof TypeId
 
-/**
- * @since 1.0.0
- * @category type ids
- */
-export const PlanTypeId: unique symbol = Symbol.for("@effect/ai/Plan")
-
-/**
- * @since 1.0.0
- * @category type ids
- */
-export type PlanTypeId = typeof TypeId
-
 /**
  * @since 1.0.0
  * @category models
  */
-export interface AiModel<in out Provides, in out Requires> extends AiPlan.AiPlan<unknown, Provides, Requires> {
+export interface AiModel<in out Provides, in out Requires>
+  extends Layer.Layer<Provides, never, Requires>, Effect.Effect<Provides & Layer.Layer<Provides>, never, Requires>
```

Comment:
```suggestion
  extends Layer.Layer<Provides, never, Requires>, Effect.Effect<Layer.Layer<Provides>, never, Requires>
```

Was the intersection required?